src/domain/Race.js

```diff
@@ -0,0 +1,36 @@
+class Race {
+  static ROUNDS = 5;
+
+  #cars;
+
+  constructor(cars) {
+    this.#cars = cars;
+  }
+
+  playRound() {
+    this.#cars.forEach((car) => car.forward());
+    return this.#cars.map(({ name, location }) => {
+      return { name, location };
+    });
```

Comment:
```suggestion
    return this.#cars.map(car => car.value);
```
이런식으로 표현할 수 있으면 어떨까요!?